manifests/service.pp

```diff
@@ -17,6 +17,12 @@
     }
 
     foreman::dynflow::worker { 'worker': }
+
+    if $jobs_hosts_queue_worker {
+      foreman::dynflow::worker { 'worker-hosts-queue':
+        queues => 'hosts_queue'
```

Comment:
If i'm reading [this](https://github.com/theforeman/puppet-foreman/blob/master/manifests/dynflow/worker.pp#L7) right, `queues` should be an array of strings

Reply:
Good catch, updated